### DISCUSSION TITLE
Layout : insert relation tables in tabs

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#3278] [Help] [answered]
Hello,

is it possible to move the relations fields/tables into a tabs in the form ?
If yes, how ?

Thanks you

JM

**Comment by danharrin** [comment, marked as answer]
The relation managers? No, they need to be below the form, HTML does not allow nesting `<form>` elements.

**Comment by Grazulex** [comment]
Ok, thanks
